.storybook/Banner.js

```diff
@@ -85,6 +85,16 @@ storiesOf('Banner', module)
         bg="lightRed"
         onClose={action('closed')}
       />
+      <Banner
+        textAlign="left"
```

Comment:
This is causing "Warning: Failed prop type: The Box `align` prop will deprecated. Please use Text instead"  Think it could be cleaned up in the PR as well?

Reply:
Good idea.  I've removed it by changing `Box` to `Text`.